### PR TITLE
Feature/serviceproviders uniquelinks

### PR DESCRIPTION
**Pull request by jvillalp**
# Description

- all servce btns that display should display corresponding link
- removed underline of link when on hover (did some styling)
- changes expansion panel to have a better flow on the movie card
 - some styling on movie card and service btns
- fixed bug/issue #137 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [x] Complete, but not tested (may need new tests)

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/movies/MovieCard.js [hunk 1]
```diff
@@ -228,49 +229,47 @@ const useStyles = makeStyles((theme) => ({
...
-  // margin:'auto',
-  // width:'80%',
-  //   backgroundColor: '#212120',
-
-  // },
+  expansionPanal: {
+    dislay: 'flex',
+    justifyContent: 'center',
+    margin: 'auto',
+    // width: '80%',
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:253]: delete?

File: src/components/movies/MovieCard.js [hunk 2]
```diff
@@ -228,49 +229,47 @@ const useStyles = makeStyles((theme) => ({
...
+    background: "rgb(23, 23, 23, .96)",
+    boxShadow: theme.shadows[5],
+
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:256]: delete blank line?

File: src/components/movies/MovieCard.js [hunk 3]
```diff
@@ -543,9 +542,9 @@ function MovieCard({
                     )}
                   </CardActions>
                 ) : (
-                  // </div>
-                  ""
-                )}
+                    // </div>
+                    ""
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:546]: delete?

Comment (msinnema33) [inline src/components/movies/MovieCard.js:546]: delete commented out div

File: src/utils/axiosWithAuth.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const axiosWithAuth = (accessToken) => {
         headers: {
             Authorization: `Bearer ${accessToken}`
         },
-        baseURL: baseURL
+        baseURL: 'http://localhost:5000/api/users'
     })
```
Comment (msinnema33) [inline src/utils/axiosWithAuth.js:12]: change to baseURL env file entry??

**Comment by jvillalp** [comment]
> several lines of commented out code (in line comments). Need to change baseURL.

will do! good catch!

**Comment by Steven-matos** [comment]
this PR is going to cause merge conflicts with my PR request

**Comment by jvillalp** [comment]
> this PR is going to cause merge conflicts with my PR request

did you already merge yours to master? we'll need to do that with either this one or the other, depending on which one gets added first.

**Comment by Steven-matos** [comment]
> > this PR is going to cause merge conflicts with my PR request
> 
> did you already merge yours to master? we'll need to do that with either this one or the other, depending on which one gets added first.

mine is waiting to be added I cleaned out a lot of code in the movie card that's why I say its going to cause conflict

File: src/components/movies/MovieCard.js (hunk 3 above)
Comment (jvillalp) [reply src/components/movies/MovieCard.js:538]: still working on this one

**Comment by jvillalp** [comment]
> > > this PR is going to cause merge conflicts with my PR request
> > 
> > 
> > did you already merge yours to master? we'll need to do that with either this one or the other, depending on which one gets added first.
> 
> mine is waiting to be added I cleaned out a lot of code in the movie card that's why I say its going to cause conflict

some of the code you deleted i actually uncommented for the 'where to watch' btn, so that might be an issue, but if you merge yours to master, i can bring those changes back and that should mess with ur edits right? since you just worked on the movie slider for the ratings page? my edits are more geared towards the service providers button/links

**Comment by moviedatascience** [comment]
I'm going to merge just to see what it looks like deployed but would love for Mike to do another check to make sure it doesn't have errant code lines.

**Pull request merged**